tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h

```diff
@@ -579,19 +587,22 @@ class MklMatMulPrimitive : public MklPrimitive {
 
   ~MklMatMulPrimitive() {}
 
-  void Execute(const T* a_data, const T* b_data, T* c_data,
+  void Execute(const Tlhs* a_data, const Trhs* b_data, Toutput* c_data,
```

Comment:
Let's consolidate the implementation by making this function call
```Execute(a_data, b_data, c_data, nullptr, nullptr, stream);```
 instead.

Reply:
Done